hbase_storage/hbase1.1.1/src/main/java/com/splicemachine/access/hbase/HBaseConnectionFactory.java

```diff
@@ -246,6 +246,19 @@ public boolean createSpliceHBaseTables(){
                         com.splicemachine.si.constants.SIConstants.SEQUENCE_TABLE_NAME+" created");
             }
 
+            if(!admin.tableExists(TableName.valueOf(namespace, HConfiguration.MASTER_SNAPSHOTS_TABLE_NAME))){
+                HTableDescriptor td=generateNonSITable(HConfiguration.MASTER_SNAPSHOTS_TABLE_NAME);
+                admin.createTable(td);
+                SpliceLogUtils.info(LOG,
+                        HConfiguration.MASTER_SNAPSHOTS_TABLE_NAME + " created");
+            }
+
+            if(!admin.tableExists(TableName.valueOf(namespace, HConfiguration.SLAVE_REPLICATION_PROGRESS_TABLE_NAME))){
```

Comment:
Is it possible to keep this info in-memory too? It's just a mapping from regionId to LSN, right?

Reply:
@dgomezferro I agree we should rethink how to keep these information and process them more efficiently. How about file another Jira as an improvement?

Reply:
That'd be fine, yes.